src/SFA.DAS.Authorization.EmployerFeatures/Models/EmployerFeatureToggle.cs

```diff
@@ -12,7 +12,7 @@ public class EmployerFeatureToggle : FeatureToggle
 
         public bool IsUserWhitelisted(long accountId, string userEmail)
         {
-            return Whitelist.Any(w => w.AccountId == accountId && (w.UserEmails == null || w.UserEmails.Count == 0 || w.UserEmails.Contains(userEmail, StringComparer.InvariantCultureIgnoreCase)));
+            return Whitelist.Any(w => (w.AccountId == accountId || w.AccountId == null) && (w.UserEmails == null || w.UserEmails.Count == 0 || w.UserEmails.Contains(userEmail, StringComparer.InvariantCultureIgnoreCase)));
         }
```

Comment:
Any passed in account id will return true for any e-mail in the config if the account id in the config is null. Is this desired behaviour?

Reply:
@MichaelYoung1981  Can we confirm this behaviour is correct? I'm not sure where the code that uses this in Non-Levy you mentioned previously?

Reply:
Yep, that's the way I expect it to work.  If there is no account id in the config then all account ids will pass.